lib/Api/DevCycleClient.php

```diff
@@ -1,48 +1,27 @@
 <?php
 
-/**
- * DevCycleClient
- * PHP version 7.3
- *
- * @category Class
- * @package  DevCycle
- * @author   OpenAPI Generator team
- * @link     https://openapi-generator.tech
- */
-
-/**
- * DevCycle Bucketing API
- *
- * Documents the DevCycle Bucketing API which provides and API interface to User Bucketing and for generated SDKs.
- *
- * The version of the OpenAPI document: 1.0.0
- * Generated by: https://openapi-generator.tech
- * OpenAPI Generator version: 5.3.0
- */
-
-/**
- * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
- * https://openapi-generator.tech
- * Do not edit the class manually.
- */
-
 namespace DevCycle\Api;
 
 use DevCycle\Model\DevCycleEvent;
 use DevCycle\Model\DevCycleUser;
 use DevCycle\Model\DevCycleUserAndEventsBody;
+use DevCycle\Model\ErrorResponse;
```

Comment:
Probably not needed for this PR, but I think we should try and split off a bunch of this file into smaller parts, maybe moving the request parts?

Reply:
I tried to start doing that by pulling the code into functions; definitely something we want to do in the long run.